File: src/reader/shared.js
export const UNION_RAW_DATA_DEX_AG_ONLY = `with 
      raw_data as (
        SELECT *, 'ocean' as name
        FROM ocean
        UNION ALL 
        SELECT *, 'oneinch' as name
        FROM oneinch
        UNION ALL
        SELECT *, 'zeroex' as name
        FROM zeroex
        UNION ALL
        SELECT *, 'paraswap' as name
        FROM paraswap
        ),`;
export const UNION_RAW_DATA = `with 
      pre_raw_data as (
        SELECT *, 'ocean' as name
        FROM ocean
        UNION ALL 
        SELECT *, 'oneinch' as name
        FROM oneinch
        UNION ALL
        SELECT *, 'zeroex' as name
        FROM zeroex
        UNION ALL
        SELECT *, 'paraswap' as name
        FROM paraswap
        UNION ALL
        SELECT *, 'cowswap' as name
        FROM cowswap
        ), `;
export const UNION_RAW_DATA_HIGH_GAS = UNION_RAW_DATA + ` raw_data as (
        select * from pre_raw_data prd inner join  parameters pa on pa.uid = prd.uid
        where gas_price > 30
      ),`;
export const UNION_RAW_DATA_LOW_GAS = UNION_RAW_DATA + ` raw_data as (
        select * from pre_raw_data prd inner join parameters pa on pa.uid = prd.uid
        where gas_price < 30
      ),`;
export const COLORS = [
  "#345C7D",
  "#F7B094",
  "#F5717F",
  "#F7B094",
  "#6C5B7A",
];

File: evaluation_dashboards/app/src/shared.py
UNION_RAW_DATA_JOINED_WITH_PARAMETER = """with raw_data as (
        select pa.uid, sell_amount, buy_amount, executed_sell_amount, executed_buy_amount, exchange, name, data, output_value_usd, gas_cost_usd_from_trader_contract, gas_cost_usd_from_trace_callMany, gas_price, sell_token_price, buy_token_price, eth_price, block_number from exchange prd inner join parameters pa on pa.uid = prd.uid 
          where output_value_usd <> 'NaN' and (executed_sell_amount != 0 or name='cowswap')
          ORDER BY block_number DESC limit 10000
      ),"""
# Since reverted transactions have an executed_sell_amount of zero, we will filter them out. Only cowswap has executed_sell_amount of zer0, as the tx can not be simulated
